app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split()
```

Comment:
```suggestion
    command_type, source, destination = command.split()
```
it's more readable to use unpacking here